tests/v2/validation/terratest/config.go

```diff
@@ -0,0 +1,78 @@
+package terratest
+
+type GoogleAuthEncodedJSON struct {
+	AuthProviderX509CertURL string `json:"auth_provider_x509_cert_url" yaml:"auth_provider_x509_cert_url"`
+	AuthURI                 string `json:"auth_uri" yaml:"auth_uri"`
+	ClientEmail             string `json:"client_email" yaml:"client_email"`
+	ClientID                string `json:"client_id" yaml:"client_id"`
+	ClientX509CertURL       string `json:"client_x509_cert_url" yaml:"client_x509_cert_url"`
+	PrivateKey              string `json:"private_key" yaml:"private_key"`
+	PrivateKeyID            string `json:"private_key_id" yaml:"private_key_id"`
+	ProjectID               string `json:"project_id" yaml:"project_id"`
+	TokenURI                string `json:"token_uri" yaml:"token_uri"`
+	Type                    string `json:"type" yaml:"type"`
+}
+
+type Nodepool struct {
+	Quantity         int64  `json:"quantity" yaml:"quantity"`
+	Etcd             bool   `json:"etcd" yaml:"etcd"`
+	Cp               bool   `json:"cp" yaml:"cp"`
```

Comment:
[ease-of-use enhancement]:
not a fan of abbreviating node pool roles. can we update this to the actual names? also that's what we use in other tests, so we should be consistent.

Reply:
Agreed, the names should be the actual names and not abbreviations.

Reply:
heard.  No problem, i'll get these values updated